packages/ripple-address-codec/src/utils.ts

```diff
@@ -19,32 +19,32 @@ export function seqEqual(arr1: Sequence, arr2: Sequence): boolean {
   return true
 }
 
-/**
- * Check whether a value is a sequence (e.g. array of numbers).
- *
- * @param val The value to check.
- */
+// .
```

Comment:
Why is there a period at the top here?

Reply:
The linter adds periods to the end of lines - there's probably a space in this line, so the linter assumes there needs to be a period. The line should just be deleted.